test/timestamps.test.js

```diff
@@ -1026,6 +1026,30 @@ describe('timestamps', function() {
     assert.equal(doc.sections.length, 1);
     assert.equal(doc.sections[0].title, 'h1');
   });
+
+  it('findOneAndUpdate creates subdocuments with timestamps in reverse order', async function() {
```

Comment:
I noticed the test cases listed here include the Github issue (e.g. gh-12475).

Also, wouldn't we describe this as: 
> findOneAndUpdate creates subdocuments with timestamps in the expected order

At least, that's how I'm reading the `expected` variable.